### PR TITLE
Add ETH and ETH test chain in ChainId and disable entities tests

### PR DESCRIPTION
**Pull request by medardm**
- will fix the entities tests in the future

References #1

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -5,7 +5,9 @@ export type BigintIsh = JSBI | bigint | string
 
 export enum ChainId {
   MAINNET = 56,
-  BSCTESTNET = 97
+  BSCTESTNET = 97,
+  ETHMAINNET = 1,
+  ETHTESTNET = 1,
```
Comment (mativanarquero) [inline src/constants.ts]: Kuys same po talaga ETH test net chain id sa mainnet? What if i **5** po natin sa testnet for goerli?

Comment (medardm) [reply src/constants.ts]: aw haneps hahaha

**Review by mativanarquero — APPROVED** [review]
LGTM

**Review by swengr-janan — APPROVED** [review]
Added!

**Review by aldrickb — APPROVED** [review]

**Pull request merged**